yb-voyager/cmd/common.go

```diff
@@ -1269,6 +1269,148 @@ func (ar *AssessmentReport) GetClusterSizingRecommendation() string {
 		ar.Sizing.SizingRecommendation.MemoryPerInstance, ar.Sizing.SizingRecommendation.EstimatedTimeInMinForImport)
 }
 
+func (ar *AssessmentReport) GetTotalTableRowCount() int64 {
```

Comment:
@priyanshi-yb pls review these. I think Sanyam has just moved them around, so shouldn't really be any logic change.

Reply:
yeah, removed the SizeDetails struct to reduce the number of components in code.